fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -198,7 +198,7 @@
     @Nonnull private Config config;
     private int configLoaderInvocationCount = 0;
 
-    private final IndexFromIndexRequest indexFromIndex;
+    @Nonnull private final IndexFromIndexRequest indexFromIndex;
```

Comment:
You will need to set a default value in the builder to make it an nonnull.

Reply:
Shouldn't line 243 in the constructor handle this? 
`  this.indexFromIndex = indexFromIndex != null ? indexFromIndex : new IndexFromIndexRequest(null);`

Reply:
You can do it at https://github.com/FoundationDB/fdb-record-layer/pull/1079/files#r541144354 where all default values live